### ISSUE TITLE
Control.Exception block and unblock were removed from GHC (Head)

### ISSUE THREAD

**Issue by jcristovao** [skogsbaer/hscurses#17]
Hi,

I was trying out GHC Head (7.7), and was unable to compile hscurses:

``` Haskell
[1 of 7] Compiling UI.HSCurses.MonadException ( UI/HSCurses/MonadException.hs, dist/build/UI/HSCurses/MonadException.o )

UI/HSCurses/MonadException.hs:107:20:
    Not in scope: ‛block’
    Perhaps you meant ‛blockM’ (line 29)

UI/HSCurses/MonadException.hs:108:20:
    Not in scope: ‛unblock’
    Perhaps you meant ‛unblockM’ (line 30)
```

It seems `block` and `unblock` were already deprecated in 7.6.3.
I think `block` could be easily replaced by `mask_`, but I'm not so sure about `unblock`.
